src/test/python/deep_qa_test/layers/backend/batch_dot_test.py

```diff
@@ -0,0 +1,32 @@
+# pylint: disable=no-self-use,invalid-name
+import numpy
+from numpy.testing import assert_almost_equal
+import keras.backend as K
+
+from deep_qa.layers.backend.batch_dot import BatchDot
+
+
+class TestMaskedBatchDotLayer:
+    def test_compute_mask(self):
+        tensor_a = K.variable(numpy.array([[[0.3, 0.1], [0.4, 0.2], [0.8, 0.1]],
```

Comment:
You can just do `numpy.random.rand(shape)`; it would make this a little easier to read, and easier to write more tests.